src/main/webapp/about.html

```diff
@@ -0,0 +1,224 @@
+<!DOCTYPE html>
+<!--
+To change this license header, choose License Headers in Project Properties.
+To change this template file, choose Tools | Templates
+and open the template in the editor.
+-->
+<html>
+    <head>
+        <title>About Us</title>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <style>
+            
+            .wrapper{
```

Comment:
Same here. Wouldn't it be cleaner with style separately? idk exactly how jsp works when it comes to this